content/en/tracing/manual_instrumentation/dotnet.md

```diff
@@ -51,11 +85,104 @@ catch(Exception e)
 }
 ```
 
+This will set 3 tags on the span: `"error.msg":exception.Message`,  `"error.stack":exception.ToString()`, and `"error.type":exception.GetType().ToString()`.
+
+### Manually creating a new span
+
+Customize your observability by programmatically creating spans around any block of code.  Spans created in this manner integrate with other tracing mechanisms automatically. In other words, if a trace has already started, the manual span will have its caller as its parent span. Similarly, any traced methods called from the wrapped block of code will have the manual span as its parent.
+
+```csharp
+using (var parentScope =
+       Tracer.Instance.StartActive("manual.sortorders"))
+{
+    using (var childScope =
+           Tracer.Instance.StartActive("manual.sortorders.child"))
+    {
+        // Nest using statements around code you would like to trace
+        SortOrders();
+    }
+}
+```
+
+<br>
+
+## Resource Filtering for Health Checks & other endpoints
+
+The Agent can be configured to exclude a specific Resource from Traces sent by the Agent to the Datadog application. To prevent the submission of specific Resources, use the `ignore_resources` setting in the `datadog.yaml` file . This setting enables the creation of a list containing one or more regular expressions, which instructs the Agent to filter out Traces based on their Resource name.
+
+If you are running in a containerized environment, set `DD_APM_IGNORE_RESOURCES` on the container with the Datadog Agent instead. To learn more, [click here][5].
+
+This can be useful for excluding any Health Checks or otherwise simulated traffic from the calculation of metrics for your services.
+```text
+## @param ignore_resources - list of strings - optional
+## A blacklist of regular expressions can be provided to disable certain traces based on their resource name
```

Comment:
```suggestion
## A blacklist of regular expressions used to disable certain traces based on their resource name.
```

Reply:
Should we avoid the use of "blacklist" here and instead use "exclude". Something like

```
A list of regular expressions used to exclude certain traces based on their resource name.
```

Reply:
+1 for exclude